<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit_Raio">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_Raio">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Raio</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Resultado">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>240</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Resultado</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Executar">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>150</y>
      <width>112</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Executar</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_Resultado">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>260</y>
      <width>641</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_FiguraGeometrica">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Figura Geométrica</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Esfera">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>100</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Esfera</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Cilindro">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>100</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Cilindro</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Cone">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>100</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Cone</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Altura">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>170</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_Altura">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>150</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Altura</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Selecione uma opção:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Limpar">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>150</y>
      <width>112</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Limpar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
